cmd/tiller/tiller.go

```diff
@@ -57,6 +58,10 @@ const (
 
 	probeAddr = ":44135"
 	traceAddr = ":44136"
+
+	// defaultMaxHistory sets the maximum number of releases to 0: unlimited
```

Comment:
- [ ] Move this comment down one line to match up with the constant it's describing.
- [ ] Consider rewording it slightly:  
  > defaultMaxHistory allows an unlimited number of versions per release.